WordPress/Classes/Networking/ServiceRemoteREST.m

```diff
@@ -1,6 +1,10 @@
 #import "ServiceRemoteREST.h"
 #import "WordPressComApi.h"
 
+static NSString* const ServiceRemoteRESTApiVersionStringInvalid = @"invalid_api_version";
+static NSString* const ServiceRemoteRESTApiVersionString_1_1 = @"v1.1";
+static NSString* const ServiceRemoteRESTApiVersionString_1_2 = @"v1.2";
```

Comment:
This is OK, but just FYI, when you declare a `static` constant like this, it's only defined for the current file so there's no need to use a prefix to namespace it. This one could be just `ApiVersionString_1_2`